app/src/main/java/com/jerboa/ui/components/person/PersonProfile.kt

```diff
@@ -278,6 +289,7 @@ fun PersonProfileMoreDropdown(
     onBlockPersonClick: () -> Unit,
     onReportPersonClick: () -> Unit,
     onMessagePersonClick: () -> Unit,
+    openMatrix: (() -> Unit)?,
```

Comment:
Hrm.... not a fan of optional params, as they often result in places you don't catch them. Probably best would be 

```
hasMatrix: Boolean,
onOpenMatrixClick: () -> Unit
```

Reply:
Optional params have indeed been a footgun but this param is not optional. You are forced to pass null or a lambda. Not doing will result in a compile error. *That would not be the case if I did `openMatrix: (() -> Unit)? = null`*

Reply:
mmmk cool.